UPGRADING.rst

```diff
@@ -80,6 +80,40 @@ Manually deploy the ``shared`` component of any main deployment just before
 pushing the merge commit to the GitLab instance in that deployment.
 
 
+#4673 Eliminate burner accounts
+===============================
+
+When this PR lands develop, in `platform-hca-dev` Google cloud project, the
+operator should perform the following steps, and then announce for other
+developers to do the same. Note that "your burner" account is the one where your
+CruzID is followed by `….ucsc.edu@gmail.com`.
+
+#) Remove your burner account from the system's Google Cloud project:
+
+    #) Go to the Google Cloud console, select the `platform-hca-dev` project,
+       and navigate to ``IAM & Admin`` -> ``IAM``
+
+    #) Select your individual burner account; it does not include the string
+       "azul"
+
+    #) Click ``DISABLE SERVICE ACCOUNT`` -> ``DISABLE``
+
+    #) Click ``DELETE SERVICE ACCOUNT`` -> ``DELETE``
+
+#) Close your burner Google account:
+
+    #) Signin to Google using your burner account, in the icon with your
+       burner's name initial (upper right hand of the page), click the
+       ``Manage your Google Account`` button. And navigate to ``Data & Privacy``
+
+    #) At the bottom of the page, in ``More options``, click on the
+       ``Delete your Google Account`` option, complete Google's requisites and
+       terminate your burner account by clicking on ``Delete Account``
+
+#) Make sure to registere your account with SAM as `described
+<./README.md#234-google_cloud,_tdr_and_sam>`_ in our README.
+
+
```

Comment:
```suggestion

#) Close your burner Google account:

    #) Sign in to Google using your burner account. Click on the icon with your
       burner's name initial (upper right hand of the page), click the
       ``Manage your Google Account`` button, and navigate to ``Data & Privacy``

    #) At the bottom of the page, under ``More options``, click on the
       ``Delete your Google Account`` button. Complete Google's requisites and
       terminate your burner account by clicking on ``Delete Account``

#) Make sure to register your UCSC account with SAM as `described
<./README.md#234-google_cloud,_tdr_and_sam>`_ in our README.


```

Reply:
I updated the section. It's necessary to remove the burner from the Google project.